easier68k/core/util/find_module.py

```diff
@@ -11,7 +11,22 @@
     'easier68k.core.opcodes.lea.Lea',
     'easier68k.core.opcodes.trap.Trap',
     'easier68k.core.opcodes.opcode_or.Or',
-    'easier68k.core.opcodes.add.Add'
+    'easier68k.core.opcodes.add.Add',
+    'easier68k.core.opcodes.Bcc.BRA',
```

Comment:
Sorry that this comment was made so late, but I think we should follow the pattern that filenames are lowercase, and classnames are CamelCase.

`opcode_or` was an exception because of the keyword `or`